app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    _ = [Person(human["name"], human["age"]) for human in people]
+
+    for human in people:
+        if human.get("wife"):
+            Person.people[human["name"]].wife = Person.people[human["wife"]]
+        elif human.get("husband"):
+            Person.people[human["name"]
+                          ].husband = Person.people[human["husband"]]
```

Comment:
bad formating :(

Reply:
It was auto-formatting in my editor. I think I need to consider disabling this feature or switch to PyCharm)

Reply:
Can you suggest a better  auto-formatting extension for Visual Studio Code. If u know?